src/main/java/uk/gov/pay/api/filter/ratelimit/RateLimiter.java

```diff
@@ -68,4 +70,21 @@ void auditRateOf(String key) {
             LOGGER.error("Unexpected error creating a Rate Limiter object in cache", e);
         }
     }
+
+    private int getRateForMethod(String method){
+        if(METHOD_POST.equals(method)){
```

Comment:
use `javax.ws.rs.HttpMethod.POST.equals(method)`

Reply:
changed to HttpMethod.POST